core/src/executor/evaluate/function.rs

```diff
@@ -538,6 +540,21 @@ pub fn is_empty<'a>(expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
     Ok(Evaluated::from(Value::Bool(length == 0)))
 }
 
+pub fn values<'a>(expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
+    let expr: Value = expr.try_into()?;
+    match expr {
+        Value::Map(m) => {
+            let mut values = Vec::new();
+            // list will be sorted by key value that is String type.
+            for key in m.keys().sorted() {
+                values.push(m[key].clone());
+            }
+            Ok(Evaluated::from(Value::List(values)))
+        }
```

Comment:
```suggestion
        Value::Map(m) => Ok(Evaluated::from(Value::List(
            m.into_iter()
                .sorted_by(|(k1, _), (k2, _)| k1.cmp(k2))
                .map(|(_, v)| v)
                .collect(),
        ))),
```

You can clear the `clone` logic by using ownership of the hashmap, sorting using `sorted_by`, and doing `collect` with the results.